controller/controller.js

```diff
@@ -0,0 +1,12 @@
+const {fetchCategories} = require('../model/model')
+//handling sql queries and directing them to an output in the controller
+
+
+
+exports.getCategories = (req, res) => {      //takes category data defined in the model and sends it through
+  console.log('found the controller')
```

Comment:
once finished debugging, remember to remove console.logs

Reply:
Question, am I alright in adding comments to guide me about MVC ordering inside the model/controller?

Reply:
Yes this is fine, but once you have that task completed, they need to be deleted.